haskell/assets/ghci_script

```diff
@@ -1,2 +1,2 @@
-:add {ADD_SOURCES}
+:load {ADD_SOURCES}
```

Comment:
The only concern would be related to https://github.com/tweag/rules_haskell/pull/1858

> Since GHC version 9.4.1 a `:reload` in the GHCi script triggers a secondary recompilation of the modules, effectively doubling GHCi start-up times.

Does `:load` instead of `:add` here affect REPL startup time?

Reply:
I tested with a non-trivial servant project first with `:add` and with `:load` afterwards, like this
```
time bazel run //server/src:server@ghci <<<:quit
```
I got wall times from 3.3 to 4.3 seconds, for both methods. So there is some variation for both, but the times are definitely in the same ball park.